src/inject.js

```diff
@@ -136,6 +136,9 @@ const stringHandlers = {
 // set of class names)
 let alreadyInjected = {};
 
+// The number of classes that have been injected
+let classUIDCount = 0;
```

Comment:
Yeah, I think that hashing that combined class name is the best approach here. (you probably can just join it, I don't think that adding in some random `-o_O-`s is improving our hash value)